frontend/src/scenes/surveys/SurveyStatsSummary.tsx

```diff
@@ -159,24 +160,30 @@ function SurveyStatsStackedBar({
 
 function SurveyStatsContainer({ children }: { children: React.ReactNode }): JSX.Element {
     const { filterSurveyStatsByDistinctId, processedSurveyStats, survey } = useValues(surveyLogic)
-    const { setFilterSurveyStatsByDistinctId } = useActions(surveyLogic)
+    const { setFilterSurveyStatsByDistinctId, updateSurvey } = useActions(surveyLogic)
+
+    const isPubliclyShareable = !!survey.is_publicly_shareable
 
     return (
         <div className="flex flex-col gap-1">
             <div className="flex items-center gap-2 justify-between">
                 <h3 className="mb-0">Survey performance</h3>
-                {processedSurveyStats && processedSurveyStats[SurveyEventName.SHOWN].total_count > 0 && (
-                    <div className="flex items-center gap-2">
-                        <LemonLabel>
-                            Count each person once
-                            <LemonSwitch
-                                checked={filterSurveyStatsByDistinctId}
-                                onChange={(checked) => setFilterSurveyStatsByDistinctId(checked)}
-                                tooltip="If enabled, each user will only be counted once, even if they have multiple responses."
-                            />
-                        </LemonLabel>
-                    </div>
-                )}
+                <div className="flex items-center gap-2">
+                    {isPubliclyShareable && <CopySurveyLink surveyId={survey.id} />}
+                    <LemonSwitch
+                        checked={isPubliclyShareable}
+                        onChange={(checked) => updateSurvey({ is_publicly_shareable: checked })}
+                        label="Collect responses via external link"
+                    />
```

Comment:
will this be visible for every presentation type? eg popver, widget and API? should that be the case?

Reply:
talked more about it in the discussions below, but will change so it instead is its own type